_posts/2023-09-10-python-multiple-arithmetic-operators.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "The surprising behavior of multiple + and - operators in Python"
+header:
+  image: /assets/images/headers/2023-09-28_pythons-weird-arithmetic.png
+  image_description: "An example of Python's weird arithmetic"
+  teaser: /assets/images/teasers/2023-09-28_pythons-weird-arithmetic.png
+excerpt: Let's explore this strange quirk of the Python language and learn from it.
+---
+
+All programming languages have some quirks and strange behavior.
+Python is certainly no exception.
+I was however quite surprised recently when I found out that Python allows multiple `+` and `-` in a row.
+Let's explore this behavior and learn about order of operation from it.
+
+# Use as many `+` and `-` together as you want
```

Comment:
I like the intro, but maybe could add a hook? Something to get me interested to learn more

Maybe show the weird syntax, pose a question, then answer it in the article. People will understand the syntax, only nerds gonna be reading